test/internal/k8sutils/utils.go

```diff
@@ -373,3 +374,8 @@ func NamespaceExists(ctx context.Context, clientset *kubernetes.Clientset, names
 	}
 	return true, nil
 }
+
+// return podLabelSelector and nodeLabelSelector
+func CreateLabelSelectors(podLabelKey, nodepoolKey string, podPrefix, nodepoolSelector *string) (string, string) {
```

Comment:
Can we change this to take in one set of inputs for a single output? Will allow for more re-use.